sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/BuilderHelper.java

```diff
@@ -89,17 +96,15 @@ static HttpPipeline buildPipeline(
             policies.add(credentialPolicy);
         }
 
-        //4
-        policies.addAll(additionalPolicies);
+        // Add per retry additional policies.
+        policies.addAll(perRetryAdditionalPolicies);
         HttpPolicyProviders.addAfterRetryPolicies(policies); //should this be between 3/4?
```

Comment:
For the comment here, I think no as these are added via a service provider so the customer passed policies should get precidence.

Reply:
Was referring to the Javadoc comment: `//should this be between 3/4?`

Reply:
Oh perfect, thank you!